extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -213,29 +215,14 @@ func CreateRKE2K3SSnapshot(client *rancher.Client, clusterName string) error {
 }
 
 // RestoreRKE1Snapshot is a helper function to restore a snapshot on an RKE1 cluster. Returns error if any.
-func RestoreRKE1Snapshot(client *rancher.Client, clusterName string, snapshotRestore *management.RestoreFromEtcdBackupInput, initialControlPlaneValue, initialWorkerValue string) error {
+func RestoreRKE1Snapshot(client *rancher.Client, clusterName string, snapshotRestore *management.RestoreFromEtcdBackupInput, restoreSnapshotResp *management.Cluster) error {
```

Comment:
the new name is a little confusing to me, isn't this just a cluster management object?

Reply:
Yeah, it's just restoring the snapshot. I couldn't think of a good name at the time. Any ideas?

Reply:
Resolved, just changed it to `cluster`.